_sass/blocks/_revenue.scss

```diff
@@ -4,19 +4,182 @@
 //
 // Styleguide blocks.revenue
 
-section.revenue {
+.revenue-section {
+  $dark-blue: #4b9bbf;
+  $positive-fill: $pale-blue;
+  $negative-fill: #bbb;
+
+  .filters {
+    color: $dark-blue;
+  }
+
+  button.toggle {
+    background: transparent;
+    // border: 1px solid;
+    color: $dark-blue;
+    display: block;
+    font: inherit;
+    margin: 0 auto;
+    padding: 0;
+    position: relative;
+    text-align: left;
+    width: 100%;
+
+    &:after {
+      content: $fa-var-chevron-down;
+      font-family: FontAwesome;
+      font-size: 1.2em;
+      line-height: 1;
+      margin-top: -0.5em;
+      position: absolute;
+      right: 0.5em;
+      top: 50%;
+    }
+
+    &[aria-expanded=true] {
+      margin-top: 1em;
+    }
+
+    &[aria-expanded=true]:after {
+      content: $fa-var-chevron-up;
+    }
+  }
+
+  [aria-expanded=false] {
+    display: none;
+  }
+
+  [class^=slab] {
+    padding-bottom: 1em;
+    padding-top: 1em;
+  }
+
+  eiti-slider {
+    margin-top: 1em;
+  }
 
   svg[is='eiti-map'] {
+    display: block;
     height: 250px;
+    margin-bottom: 1em;
     max-height: 30vh; // 30% of the viewport height
+    width: 100%;
   }
 
   .regions .region {
-    // display: none;
+    display: none;
 
     &.active {
       display: block;
     }
   }
 
+  .map-legend {
+    display: flex;
+    margin-bottom: 1em;
+
+    .step {
+      display: block;
+      flex: 1;
+      border-bottom-style: solid;
+      border-bottom-width: 0.5em;
+      padding-left: 0.1em;
+    }
+
+    &.legend-invalid {
+      .step .label {
+        visibility: hidden;
+      }
+    }
+  }
+
+  .timeline {
+    shape-rendering: crispEdges;
+
+    line {
+      fill: none;
+      stroke: $dark-blue;
+      stroke-linecap: square;
+      stroke-width: 1;
+      vector-effect: non-scaling-stroke;
+    }
+
+    .area {
+      fill: $positive-fill;
+
+      &.negative {
+        fill: $negative-fill;
+      }
+    }
+
+    .mask {
+      fill: $mid-blue;
+      opacity: 0.5;
+
+      line {
+        stroke-width: 2;
+      }
+    }
+  }
+
+  .subregion {
+    // @include row();
+    border-top: 1px solid $mid-gray;
+    clear: both;
+    padding: 0 0.2em;
+    position: relative;
+
+  }
+
+  .subregion-name {
+    @extend .h4;
+
+    margin: 0;
+    width: 66%;
+    // border: 1px solid red;
+
+    .color-swatch {
+      display: inline-block;
+      height: 0.7em;
+      margin-right: 0.2em;
+      width: 0.7em;
+    }
+  }
+
+  .subregion-chart {
+    display: inline-block;
+    float: right;
+    min-width: 200px;
+    position: relative;
+    top: 0.4em;
+    width: 33%;
+
+    .bars {
+      display: block;
+      vertical-align: baseline;
+
+      .label {
+        font-weight: $weight-light;
+      }
+
+      .negative .label {
+        font-size: 0.8em;
+      }
+
+      .negative .bar {
+        fill: $negative-fill;
+      }
+
+      .positive .bar {
+        fill: $dark-gray;
+      }
+    }
+  }
+
+  @include respond-to(medium-down) {
```

Comment:
Expected item on line 179 to appear before line 12. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets